main.js

```diff
@@ -59,34 +61,40 @@ function addPassword(service, password) {
   passwordValue.size = 50;
   passwordValue.setAttribute('autocomplete', 'off');
   passwordCell.appendChild(passwordValue);
+  passwordValue.className = 'form-control monospace';
   row.appendChild(passwordCell);
   let copyButtonCell = document.createElement('td');
-  let copyButton = document.createElement('input');
+  let copyButton = document.createElement('button');
   copyButton.type = 'button';
   copyButton.value = 'Copy To Clipboard';
+  copyButton.className = 'btn btn-outline-primary';
+  copyButton.innerHTML = 'Copy To Clipboard  <img src="clipboard.svg" alt="">';
```

Comment:
These icons look cool! I also think we can perhaps use them as a mean to decrease the required width of the page. Let's remove the text "Copy to Clipboard" and "Remove Password" as I feel the icons are pretty self explanatory. To replace the "Copied!" alert that pops up let's instead use an [auto-hiding toast notification](https://getbootstrap.com/docs/4.2/components/toasts/) when the copy button is pressed.

Reply:
Ooh, That looks awesome! But we'll need to have jquery. I'll get it running and update my branch so you can see it in action.